databuilder/extractor/bigquery_metadata_extractor.py

```diff
@@ -117,6 +118,21 @@ def _retrieve_tables(self, dataset):
 
             for table in page['tables']:
                 tableRef = table['tableReference']
+
+                table_id = tableRef['tableId']
+                last_eight_chars = table_id[-8:]
```

Comment:
could you put an example of the target table_id? is it always something YYYYmmdd format? And could you create a constant for -8 which could be used in here and L127 as well?

Reply:
I gave an example in the code and also made it a constant.